.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
I think it's more a work-around for the fact that travis bots have a ***very*** old version of C++.  Moving over to trusty seems to be the sanctioned approach and how we're approaching it elsewhere in Dart.

cc @whesse @devoncarew